plugins/gstperf.c

```diff
@@ -401,16 +422,91 @@ gst_perf_cpu_get_load (GstPerf * perf, guint32 * cpu_load)
   } else {
     *cpu_load = 0;
   }
+
   /*Remember the total and idle CPU for the next check */
   perf->prev_cpu_total = total;
   perf->prev_cpu_idle = idle;
+
+  return TRUE;
+
+cpu_failed:
+  GST_ERROR_OBJECT (perf, "Failed to get the CPU load");
+  return FALSE;
+}
+
+#elif IS_MACOSX
+static gboolean
+gst_perf_cpu_get_load_macosx (GstPerf * perf, guint32 * cpu_load)
+{
+  guint32 idle = 0;
+  guint32 total = 0;
+  guint32 diff_total, diff_idle;
+  host_cpu_load_info_data_t cpuinfo;
+  mach_msg_type_number_t count = HOST_CPU_LOAD_INFO_COUNT;
```

Comment:
`count` is not used to calculate anything, however, it declares and defines, why?

Reply:
It is needed for the `host_statistics` call in line 453.